### PR TITLE
rh_qemu_iotests: update case steps when test  with qemu-kvm

### PR DESCRIPTION
**Pull request by pingl2016**
Skip to run configure in source code when test with qemu-kvm

Signed-off-by: Ping Li <pingl@redhat.com>

ID: 1426893

### CODE REVIEW

File: qemu/tests/rh_qemu_iotests.py [hunk 1]
```diff
@@ -118,10 +118,15 @@ def run_test(qemu_src_dir):
...
     build_root = params.get("build_root", "/root/rpmbuild")
     rpmbuild_clean_cmd = params["rpmbuild_clean_cmd"]
+    version_pattern = params["qemu_version_pattern"]
+    cmd = "%s -version" % utils_misc.get_qemu_binary(params)
+    output = process.system_output(cmd, shell=True)
+    qemu_version = re.search(version_pattern, output, re.M).group(0)
     cwd = os.getcwd()
     (qemu_src_dir, spec) = install_test(build_root)
     try:
-        config_test(qemu_src_dir)
```
Comment (xutian) [inline qemu/tests/rh_qemu_iotests.py]: why not 
```
if "qemu-kvm-rhev" in output:
     config_test(qemu_src_dir)
```

File: qemu/tests/cfg/rh_qemu_iotests.cfg [hunk 2]
```diff
@@ -13,6 +13,7 @@
     get_src_cmd = rpm -ivhf %s && rpmbuild -bp %s --nodeps --force
     rpmbuild_clean_cmd = rpmbuild --clean %s --nodeps
     iotests_result_pattern = "Failed\s*(\d+)\s*of\s*(\d+)\s*test"
+    qemu_version_pattern = "(?<=\()\S+(?=\))"
```
Comment (xutian) [inline qemu/tests/cfg/rh_qemu_iotests.cfg]: I think that's useless

File: qemu/tests/rh_qemu_iotests.py [hunk 3]
```diff
@@ -118,10 +118,15 @@ def run_test(qemu_src_dir):
 
     build_root = params.get("build_root", "/root/rpmbuild")
     rpmbuild_clean_cmd = params["rpmbuild_clean_cmd"]
+    version_pattern = params["qemu_version_pattern"]
+    cmd = "%s -version" % utils_misc.get_qemu_binary(params)
+    output = process.system_output(cmd, shell=True)
+    qemu_version = re.search(version_pattern, output, re.M).group(0)
```
Comment (xutian) [inline qemu/tests/rh_qemu_iotests.py]: no need to match it.

File: qemu/tests/rh_qemu_iotests.py (hunk 1 above)
Comment (pingl2016) [inline qemu/tests/rh_qemu_iotests.py]: updated, thanks

**Comment by xutian** [comment]
Acked

**Pull request merged**